source/client.go

```diff
@@ -80,6 +80,10 @@ func (c Client) New(params *stripe.SourceObjectParams) (*stripe.Source, error) {
 			}
 		}
 
+		if params.Token != "" {
+			body.Add("token", token)
+		}
```

Comment:
@antonholmquist Thanks for the contribution!

You've got a bit of a syntax problem here. I think this should be `params.Token`. Otherwise this looks good to me. Thanks!